.env-dist

```diff
@@ -2,3 +2,5 @@ DEBUG=True
 DEV=True
 ALLOWED_HOSTS=*
 SWITCH_NEWSLETTER_MAINTENANCE_MODE=False
+CSP_DEFAULT_SRC='self',*.mozilla.net,*.mozilla.org,*.mozilla.com,*.s3.amazonaws.com
+CONTENT_CARDS_URL=http://bedrock-dev-media.s3.amazonaws.com/media/
```

Comment:
Do we need to add this variable for demo deployments?

Reply:
oooooooooo... good point. I think we should point demos to https://www-dev.allizom.org/media/. Should probably use that both places... I'll do that. Good catch.